microsoft/atlas-design
# Add timeline component and thread-history pattern

Task: task-[1013232](https://ceapex.visualstudio.com/Engineering/_workitems/edit/1013232): Add timeline to atlas

Link: [preview](http://localhost:1111/)

This PR aims to add a vertical timeline component that uses the updated badge component and well as introduces the thread history pattern that is used in Q&A.

The styles and structure for the timeline component will be broad enough to be used is other instances of Learn.

## Testing

1. `git checkout jamesbe/thread-history; npm start;`
2. Navigate to the [timeline component](http://localhost:1111/components/timeline.html)
 Expected result: timelines usage should be clear
4. Navigate to the [thread history pattern](http://localhost:1111/pattenrs/thread-history.html)
  Expected result: the thread history pattern should be understandable to implement

## Contributor checklist
